### PR TITLE
Text for behave (fixes #56)

### LINKED ISSUE #56: [Behave] python behave framework

### PR DESCRIPTION
**Pull request by sseliverstov**

### CODE REVIEW

File: docs/2.0/integration/behave.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+[source, bash]
+----
+$ behave -f allure_behave.formatter:AllureFormatter -o %allure_result_folder% ./features
+----
+
+== Features
+
+=== Severity
+Tags that matched to severity names, like critical, trivial, etc, will be interpreted as feature or scenario severity.
+Scenario inherite a feature severity, the same way, scenario severity is overdride a feature severity. When situated
```
Comment (ehborisov) [inline docs/2.0/integration/behave.adoc]: Scenario inherits feature severity if not provided, or overrides it in the other case. If there is more than one severity definition tag, only the last one is used.

File: docs/2.0/integration/behave.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+more than one severity definition tag, last one is used.
+
+=== Steps and Scenarious status
+Steps with assert exceptions will be mark as a failed. All other exceptions are reason for broken status.
```
Comment (ehborisov) [inline docs/2.0/integration/behave.adoc]: Steps with assertion exceptions will be marked as `failed`. Other exceptions thrown during the test execution will cause it to have status `broken`.

File: docs/2.0/integration/behave.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Scenario has status that corresponds to first unsuccessful step status. If all steps are passed, scenario is passed.
```
Comment (ehborisov) [inline docs/2.0/integration/behave.adoc]: Scenario status will be determined by the first unsuccessful step status. When all steps are passed, then the whole scenario is considered passed.

File: docs/2.0/integration/behave.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+[source, bash]
+----
+$ behave -f allure_behave.formatter:AllureFormatter -o %allure_result_folder% ./features
+----
+
+== Features
+
+=== Severity
+Tags that matched to severity names, like critical, trivial, etc, will be interpreted as feature or scenario severity.
```
Comment (ehborisov) [inline docs/2.0/integration/behave.adoc]: Tags that are matched to severity names

**Comment by sseliverstov** [comment]
@ehborisov thanks!

File: docs/2.0/integration/behave.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+[source, bash]
+----
+$ behave -f allure_behave.formatter:AllureFormatter -o %allure_result_folder% ./features
+----
+
+== Features
+
+=== Severity
+Tags that are matched to severity names, like critical, trivial, etc, will be interpreted as feature or scenario severity.
```
Comment (ehborisov) [inline docs/2.0/integration/behave.adoc]: I also think this should go in parenthesis - "to severity names (like critical, trivial... etc.)" 
and also  "will be interpreted as a feature or scenario"

**Pull request merged**